.travis.yml

```diff
@@ -0,0 +1,20 @@
+language: java
+
+install:
+  - export PATH=/tmp/grpc-deps/bin:$PATH
+  - buildscripts/make_dependencies.sh
+
+script: ./gradlew install
```

Comment:
Wanted to be explicit about it as I think we need a multiline script now. See latest commit.